miden-tx/src/tests/mod.rs

```diff
@@ -306,28 +256,27 @@ fn executed_transaction_account_delta() {
         EXECUTION_HINT_3 = Felt::from(NoteExecutionHint::on_block_slot(1, 1, 1)),
     );
 
-    let tx_script =
-        TransactionScript::compile(tx_script_src, [], TransactionKernel::assembler_testing())
-            .unwrap();
+    let module = AccountCode::mock_module();
+
+    let tx_script = TransactionScript::compile(
+        tx_script_src,
+        [],
+        TransactionKernel::assembler_testing().with_module(module).unwrap(),
```

Comment:
Should `assembler_testing()` already be loaded with the mock module? Or does this interfere with some other tests?

Reply:
Main issue with that approach is that when trying to build the mock `AccountCode` you need to not be careful not to use the testing assembler because it will error due to duplicate definitions of the module. Because mock accounts are mainly assembled in the builders, we can use the normal assembler there though that could be a bit confusing.